extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -376,14 +376,14 @@ export class AmpIframe extends AMP.BaseElement {
    */
   updateSize_(height, width) {
     if (!this.isResizable_) {
-      user.warn(TAG_,
+      user.error(TAG_,
```

Comment:
No, iframes are supposed to be used with "tightly coupled" embeds. If they want to resize and it is plain out denied, it should be an error.

Ads should use `amp-ad`.